source/index.txt

```diff
@@ -32,145 +38,329 @@ Atlas App Services
+                       location: String!
+                     }
+
+         .. step:: Secure Your API
+            
+            Generate an API key that you can use to authenticate
+            requests. Alternatively, you can use other authentication
+            provider credentials associated with your app.
+            
+            For details, see :ref:`GraphQL API Authentication <graphql-authentication>`.
+
+         .. step:: Send Your First Request
+
+            Send your first GraphQL request from any server-side or
+            frontend environment that supports HTTPS. You can also use a
+            specialized GraphQL request tool or library.
+            
+            Learn how to :realm:`Send GraphQL Requests with Apollo
+            Client </web/graphql-apollo-react/>`.
+
+            .. tabs::
+
+               .. tab:: GraphQL
+                  :tabid: graphql
+
+                  .. code-block:: graphql
+
+                     query GetMeetingsInDateRange($start: DateTime!, $end: DateTime!) {
+                       meetings(query: {
+                         start_gte: $start,
+                         end_lte: $end
+                       }) {
+                         _id
+                         start
+                         end
+                         location
+                       }
+                     }
+
+               .. tab:: cURL
+                  :tabid: curl
+
+                  .. code-block:: bash
+
+                     curl -X POST https://realm.mongodb.com/api/client/v2.0/app/<Your App ID>/graphql \
+                       -H 'apiKey: <YOUR API KEY>' \
+                       -d {
+                         "query": "query GetMeetingsInDateRange($start: DateTime!, $end: DateTime!) { meetings(query: { start_gte: $start, end_lte: $end }) { _id start end location } }"
+                         "variables": { "start": "2022-01-01T00:00:00.000Z", "end": "2022-03-31T23:59:59.999Z" }
+                       }
+
+      .. image:: /images/plp/connection.png
+         :alt: Build a Secure GraphQL API
+
+   .. tab:: Build Event-Driven Services
+      :tabid: triggers
       
-      Learn more about the problems App Services solves - and how - to 
-      decide if App Services fits your use case.
+      .. procedure::
 
-Explore Atlas App Services
---------------------------
+         .. step:: Create and configure your Trigger
+
+            You can respond to database CRUD events and application user
+            activity as the occur or run triggers on a schedule. You can
+            also set additional conditions for when your trigger should
+            fire.
+
+         .. step:: Write the Trigger's Handler Logic
+
+            Define a serverless JavaScript function that runs when the
+            trigger fires. Or, send an event to AWS EventBridge.
+            
+            See :doc:`Trigger Function Examples </triggers/examples>`.
+
+         .. step:: Automatically Respond to Events
```

Comment:
Like the Sync tab, this isn't really an action the reader must take but a result of setting up the thing. Maybe we could reword this step title to something like "Run Your App", and then the text still works?